subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -191,7 +191,7 @@ public ExecutionResult assertOutputContains(String expectedOutput) {
 
     @Override
     public boolean hasErrorOutput(String expectedOutput) {
-        return getError().contains(expectedOutput);
+        return getError().contains(expectedOutput) || getRawError().contains(expectedOutput);
```

Comment:
This is necessary because we remove illegal access warnings from the output, right? Do we still need to do that?

Reply:
I think we still need to do that because there's also some warnings coming from Groovy: 

```
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass (file:/Users/zhb/.gradle/caches/modules-2/files-2.1/org.codehaus.groovy/groovy-all/2.4.12/760afc568cbd94c09d78f801ce51aed1326710af/groovy-all-2.4.12.jar) to method java.lang.Object.finalize()
```